pkgs/development/haskell-modules/configuration-hackage2nix/stackage.yaml

```diff
@@ -1,36 +1,36 @@
-# Stackage LTS 18.25
+# Stackage LTS 19.0
```

Comment:
@arianvp This seems to contradict your tweet: https://twitter.com/ProgrammerDude/status/1506230564923351044?s=20&t=WhBeiu9gdF78l1FeFrqKrw
(Which is good!)

Reply:
True for now, but there is some interest in tracking nightly again in the future, we'll see.

Reply:
I hope that comes to pass after the nixos-22.05 cutoff :angel:

Reply:
We generally don’t guarantee to track a certain stackage version. Our intention is to provide the most modern package set that we can without dropping any "relevant" packages. This means we aspire to track nightly, which actually means less work for us, if we can pull it off. For context it’s important to realize that we support a lot more packages than stackage. We have normally around 7000 packages working, while stackage contains about 2000.

Regardinf the current situation: When we opened this PR it was planning to switch to nightly, but then LTS 19 got released and nightly switched to ghc 9.2, so we settled for LTS again.

After this is merged we would indeed like to jump to ghc 9.2 as soon as we can, but I guess it’s unlikely that we can get it done before 22.05. We will evaluate that after the merge.